docs/dev/topology-over-raft.md

```diff
@@ -264,10 +264,11 @@ node in the cluster. The receiving node can either:
 
 The new node should not process `JOIN_NODE_RESPONSE` RPC until it is explicitly
 ordered by node that handles `JOIN_NODE_REQUEST`. This is necessary so that
-it doesn't accidentally process a join request from a previous attempt, if there
-was any. Similarly, the group 0 server must not be started on the new node
-until the `JOIN_NODE_REQUEST` RPC succeeds, otherwise it might receive some
-commands it does not understand due to unsupported features.
+it doesn't accidentally process a join response from a previous attempt, if
+there was any. Similarly, the group 0 server must not be started on the new node
+until the `JOIN_NODE_REQUEST` RPC succeeds, otherwise, if it crashes during
+the prodecude and disables some features it might receive some commands it is
+not prepared to understand.
```

Comment:
I squashed my last fix to `topology-over-raft.md` with a wrong commit. Will fix.